documentation/source/directives.rst

```diff
@@ -562,6 +562,22 @@ Config Values
    For example an enum value ``TWO = 2`` would be shown with the value 2 if this is set to ``True``,
    and without if it is set to ``False``.
 
+.. confval:: breathe_show_struct_include
```

Comment:
@rhssk Thanks for the patch, overall looks good to me.

I'm not sure about the naming though, the way things are implemented it seems like `visit_inc` is a generic function not specific to struct, although perhaps current implementations only renders this on struct?

Thoughts on renaming the option to just `breathe_show_include`?

Reply:
+1 for renaming to  ``breathe_show_include``.